test/api/models/domain/BusinessIdSpec.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api.models.domain
+
+import support.UnitSpec
+
+class BusinessIdSpec extends UnitSpec {
+
+  "BusinessId toString" should {
+    "return the BusinessId value" when {
+      "called" in {
```

Comment:
may as well remove the "when... called" layer, doesn't add anything in this instance